pages/call/[uid].tsx

```diff
@@ -65,10 +65,27 @@ export default function JoinCall(props, session) {
           height: "100%",
         },
       });
-      callFrame.join({
-        url: props.booking.dailyRef.dailyurl,
-        showLeaveButton: true,
-      });
+      callFrame
+        .join({
+          url: props.booking.dailyRef.dailyurl,
+          showLeaveButton: true,
+        })
+        .then(() => {
+          if (!props.booking.dailyRef.recordingHasStarted) {
+            callFrame.startRecording({
```

Comment:
@zach-at-the-skills - does `callFrame.startRecording` return a promise? if so, would be nice to put the fetch inside the `then` block.

Reply:
![image](https://user-images.githubusercontent.com/75326654/148972237-7bcb4456-92e4-4de6-93c2-c02ed9629cae.png)
unfortunately it doesn't return a promise. No return (void).